### ISSUE TITLE
[Bug]: 

### ISSUE THREAD

**Issue by ericrius1** [hlky/stable-diffusion-webui#540]
### What happened?

When I try to run the img2img on colab with the default image, I see the error "RuntimeError: The size of tensor a (128) must match the size of tensor b (64) at non-singleton dimension 3" in the results panel

### Version

0.0.1 (Default)

### What browsers are you seeing the problem on?

Chrome

### Where are you running the webui?

Colab

### Relevant log output

```shell

[MemMon] Recording max memory usage...

Iteration: 1/1
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/gradio/routes.py", line 247, in run_predict
    output = await app.blocks.process_api(
  File "/usr/local/lib/python3.8/site-packages/gradio/blocks.py", line 641, in process_api
    predictions, duration = await self.call_function(fn_index, processed_input)
  File "/usr/local/lib/python3.8/site-packages/gradio/blocks.py", line 556, in call_function
    prediction = await anyio.to_thread.run_sync(
  File "/usr/local/lib/python3.8/site-packages/anyio/to_thread.py", line 31, in run_sync
    return await get_asynclib().run_sync_in_worker_thread(
  File "/usr/local/lib/python3.8/site-packages/anyio/_backends/_asyncio.py", line 937, in run_sync_in_worker_thread
    return await future
  File "/usr/local/lib/python3.8/site-packages/anyio/_backends/_asyncio.py", line 867, in run
    result = context.run(func, *args)
  File "/content/stable-diffusion/scripts/webui.py", line 1288, in img2img
    output_images, seed, info, stats = process_images(
  File "/content/stable-diffusion/scripts/webui.py", line 819, in process_images
    samples_ddim = func_sample(init_data=init_data, x=x, conditioning=c, unconditional_conditioning=uc, sampler_name=sampler_name)
  File "/content/stable-diffusion/scripts/webui.py", line 1194, in sample
    xi = x0 + noise
RuntimeError: The size of tensor a (128) must match the size of tensor b (64) at non-singleton dimension 3
[MemMon] Recording max memory usage...

Iteration: 1/1
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/gradio/routes.py", line 247, in run_predict
    output = await app.blocks.process_api(
  File "/usr/local/lib/python3.8/site-packages/gradio/blocks.py", line 641, in process_api
    predictions, duration = await self.call_function(fn_index, processed_input)
  File "/usr/local/lib/python3.8/site-packages/gradio/blocks.py", line 556, in call_function
    prediction = await anyio.to_thread.run_sync(
  File "/usr/local/lib/python3.8/site-packages/anyio/to_thread.py", line 31, in run_sync
    return await get_asynclib().run_sync_in_worker_thread(
  File "/usr/local/lib/python3.8/site-packages/anyio/_backends/_asyncio.py", line 937, in run_sync_in_worker_thread
    return await future
  File "/usr/local/lib/python3.8/site-packages/anyio/_backends/_asyncio.py", line 867, in run
    result = context.run(func, *args)
  File "/content/stable-diffusion/scripts/webui.py", line 1288, in img2img
    output_images, seed, info, stats = process_images(
  File "/content/stable-diffusion/scripts/webui.py", line 819, in process_images
    samples_ddim = func_sample(init_data=init_data, x=x, conditioning=c, unconditional_conditioning=uc, sampler_name=sampler_name)
  File "/content/stable-diffusion/scripts/webui.py", line 1194, in sample
    xi = x0 + noise
RuntimeError: The size of tensor a (128) must match the size of tensor b (64) at non-singleton dimension 3
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by altryne** [comment]
I updated the colab with a "stable" checkbox, can you run it with unchecked "stable" checkbox and see if this issue returns. 

This is likely due to models not being able to generate arbitrary numbers, they have to be divided by 64

**Comment by ericrius1** [comment]
Thanks for the update! I tried it with stable box unchecked and still got the error. I'm going to do a fresh colab tomorrow and try again and post an update if something changes.

**Comment by hlky** [comment]
This is an issue with the cropping tool, specifically the resulting cropped image's resolution.
The image editor is extremely experimental and is only causing issues whereby people complain when things that could be done in an external tool (photoshop) aren't working in this project's extremely limited image editor, this project is not an image editor, it is a user interface for stable diffusion.
Please respect the limitations of the project, and refrain from complaining about issues that can be resolved simply by not expecting this webui to do them and using the appropriate tools.

**Comment by ericrius1** [comment]
Hi Hlky,
I apologized if I caused offense. I wasn't complaining and was just filing an issue I was running into. I was just using the default settings int he img2img panel and wasn't trying to crop. I'm grateful for the tool and was just trying to use the issue tracker to report issues. If there was something in the way I wrote the issue that made you feel like it was complaining, let me know so I can avoid that style of language moving forward.

Thanks
